src/train.py

```diff
@@ -140,9 +143,28 @@ def train(cfg: DictConfig) -> Tuple[dict, dict]:
 
     # initialize the model
     model: PyTorchIEModel = hydra.utils.instantiate(
-        cfg.model, _convert_="partial", **additional_model_kwargs
+        cfg.model,
+        _convert_="partial",
+        is_from_pretrained=cfg.get("pretrained_pie_model_path", None) is not None,
+        **additional_model_kwargs,
     )
 
+    if "pretrained_pie_model_path" in cfg:
+        pie_model = AutoModel.from_pretrained(cfg["pretrained_pie_model_path"])
+        loaded_state_dict = pie_model.state_dict()
+        if "pretrained_pie_model_prefix_mapping" in cfg:
```

Comment:
```suggestion
        has_prefix_mapping = cfg.get("pretrained_pie_model_prefix_mapping", None) is not None
        if has_prefix_mapping:
```
also use `not has_prefix_mapping` below to set `strict`